spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowScoreFunction.scala

```diff
@@ -55,23 +55,29 @@ class FlowScoreFunction(timeCuts: Array[Double],
             srcTopicMix: Seq[Double],
             dstTopicMix: Seq[Double]): Double = {
 
-    val FlowWords(srcWord, dstWord) = flowWordCreator.flowWords(hour: Int, minute: Int, second: Int, srcPort: Int, dstPort: Int, ipkt: Long, ibyt: Long)
 
+    val FlowWords(srcWord, dstWord) = flowWordCreator.flowWords(hour: Int, minute: Int, second: Int,
+      srcPort: Int, dstPort: Int, ipkt: Long, ibyt: Long)
 
+    val zeroProb = Array.fill(topicCount) { 0.0 }
 
-    val uniformProb = Array.fill(topicCount) {
-      1.0d / topicCount
-    }
+    /** A null value for srcTopicMix or dstTopicMix indicated the ip (source or dest respectively)
```

Comment:
This comment is very helpful, maybe it should appear in SuspiciousConnectsScoreFunction.scala as well.